### DISCUSSION TITLE
raspberry PI img

### DISCUSSION THREAD

**Discussion by p25o1** [Joshua-Riek/ubuntu-rockchip#171] [General]
hi there ,

what are the steps to  follow, to  boot a raspberry PI img on orange PI 5?

is there a way to prepare a project img to boot on orange PI 5.

thx

**Comment by Joshua-Riek** [comment]
You would need to do some serious hacking to get a raspberry pi image working on the orange pi 5. Theoretically, you must replace the kernel, device tree, and bootloader. But this would take a lot of time and can be complex if you don't know what you're doing. There could also be other issues encountered during this process. 

I would highly recommend against this as the images are meant for the raspberry pi only.

**Reply by p25o1** [reply]
thx for the reply, 

i'll use your img , and install needed apps to get started

thx again
